doc/source/train/examples/lightning/lightning_exp_tracking.ipynb

```diff
@@ -0,0 +1,285 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "(lightning_experiment_tracking)=\n",
+    "\n",
+    "# Using Experiment Tracking Tools in LightningTrainer\n",
+    "\n",
+    "W&B, CometML, MLFlow, and Tensorboard are all popular tools in the field of machine learning for managing, visualizing, and tracking experiments. The {class}`~ray.train.lightning.LightningTrainer` integration in Ray AIR allows you to continue using these built-in experiment tracking integrations.\n",
+    "\n",
+    "\n",
+    ":::{note}\n",
+    "This guide shows how to use the native [Logger](https://lightning.ai/docs/pytorch/stable/extensions/logging.html) integrations in PyTorch Lightning. Ray AIR also provides {ref}`experiment tracking integrations <tune-exp-tracking-ref>` for all the tools mentioned in this example. We recommend sticking with the PyTorch Lightning loggers.\n",
```

Comment:
Is there any scenario the user would want to use the AIR integrations instead?

Reply:
Yes. One case I can think of is when a user wants more flexible logging, our integration allows users to directly interact with (like wandb) running objects. And Lightning's logger has another layer of abstraction, so some APIs may not be supported by Lightning's logger.